src/services/device.js

```diff
@@ -9,8 +9,9 @@ const getSingleDevice = async (deviceId, user, userType, log) => {
       hospitalId: userType === 'hospital_admin' ? user._id : user.hospitalId,
       wardId: userType === 'ward_user' ? user._id : user.wardId,
     };
+    const purifyDeviceMatch = JSON.parse(JSON.stringify(deviceMatch));
```

Comment:
`purifyDevceMatch` 🤣

Reply:
i was going to do 
`let deviceMatch = JSON.parse(JSON.stringify(deviceMatch));`
if you have a better name it is so much welcomed. 🤣🤣🤣🤣

Reply:
The name is okay, it's just kinda funny.